### DISCUSSION TITLE
初始化项目无法运行

### DISCUSSION THREAD

**Discussion by Bearbearcat** [NervJS/taro#15384] [Q & A] [answered]
试着使用Taro.但是按文档来使用yarn dev:weapp显示index页面没有注册。全程没有修改过代码，完全是官方自己的代码
显示如下
![image](https://github.com/NervJS/taro/assets/37332291/f98ba90c-4855-4624-930b-528573eaa385)

**Comment by TheKonka** [comment, marked as answer]
关闭 prebundle

```js
// config/index.js
compiler: {
  type: 'webpack5',
  prebundle: {
    enable: false
  }
},
```

**Reply by Bearbearcat** [reply]
可以了 谢谢。 官方不解决这个问题吗；对于新手来讲也太不友好了。
